### PR TITLE
address issues raised by `make lint`

### LINKED ISSUE #11: Fix issue reported by `make lint`

### PR DESCRIPTION
**Pull request by sameersbn**
Closes #11

### CODE REVIEW

File: pkg/apis/messaging/v1alpha1/channel_lifecycle.go [hunk 1]
```diff
@@ -125,37 +125,37 @@ func (cs *AzureChannelStatus) MarkServiceUnknown(reason, messageFormat string, m
...
 }
 
-//MarkDispatcherFailed marks channel dispatcher ready status failed
+//MarkDispatcherFailed marks channel dispatcher ready status failed.
 func (cs *AzureChannelStatus) MarkChannelServiceFailed(reason, messageFormat string, messageA ...interface{}) {
 	ac.Manage(cs).MarkFalse(AzureChannelConditionChannelServiceReady, reason, messageFormat, messageA...)
 }
 
-//MarkDispatcherFailed marks channel dispatcher ready status failed
+//MarkDispatcherFailed marks channel dispatcher ready status failed.
```
Comment (antoineco) [inline pkg/apis/messaging/v1alpha1/channel_lifecycle.go:138]: Almost none of the Godocs above match their function name.

File: pkg/dispatcher/dispatcher.go [hunk 2]
```diff
@@ -109,15 +108,15 @@ func NewDispatcher(ctx context.Context) (*AzureDispatcher, error) {
 	return d, nil
 }
 
-//Start starts reciever
+//Start starts receiver.
```
Comment (antoineco) [inline pkg/dispatcher/dispatcher.go:111]: (nit) No space between the comment symbol and the text means that the comment is meant to be consumed by machines and not people (dummy examples: `//build`, `//codegen`).
It's mentioned somewhere in the Go wiki, can't remember where exactly.

File: pkg/apis/messaging/v1alpha1/channel_lifecycle.go [hunk 3]
```diff
@@ -94,7 +94,7 @@ func (cs *AzureChannelStatus) SetAddress(url *apis.URL) {
 }
 
 // TODO: Unify this with the ones from Eventing. Say: Broker, Trigger.
-//MarkDispatcherFailed marks channel dispatcher ready status failed
+// MarkDispatcherFailed marks channel dispatcher ready status failed.
```
Comment (antoineco) [inline pkg/apis/messaging/v1alpha1/channel_lifecycle.go:97]:
```suggestion
// PropagateDispatcherStatus uses the provided DeploymentStatus to determine
// whether the DispatcherReady condition should be marked as True or False.
```

**Comment by sameersbn** [comment]
@antoineco PTAL

**Comment by antoineco** [comment]
The comments are still inaccurate, but this could be tackled separately to avoid hijacking this PR 👍

**Pull request merged**